core/tests/test_runner/spec_test.rs

```diff
@@ -0,0 +1,58 @@
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+use super::test_file::Test;
+
+static HTML_ID_MATCHER: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r#"(<[^\s]+)[^i]*id=(?:"|')[^"']*(?:"|')([^>]*>)"#).unwrap());
```

Comment:
Uff, heavy stuff here... Is there no better alternative for this? HTML Parser and then modify actual HTML content?

Reply:
It was a fast solution to make paragraph tests pass.
Adapting this to compare sanitized HTML outputs independent of indentation and generated properties is probably worth its own PR.